modules/services/gpg-agent.nix

```diff
@@ -19,25 +19,29 @@ let
 
   inherit (config.programs.gpg) homedir;
 
-  gpgSshSupportStr = ''
-    ${gpgPkg}/bin/gpg-connect-agent --quiet updatestartuptty /bye
-  '';
+  gpgSshSupportStr = "${gpgPkg}/bin/gpg-connect-agent --quiet updatestartuptty /bye";
 
   gpgBashInitStr = ''
     GPG_TTY="$(tty)"
     export GPG_TTY
   ''
-  + optionalString cfg.enableSshSupport "${gpgSshSupportStr} > /dev/null";
+  + optionalString cfg.enableSshSupport ''
+    ${gpgSshSupportStr} > /dev/null
+  '';
 
   gpgZshInitStr = ''
     export GPG_TTY=$TTY
   ''
-  + optionalString cfg.enableSshSupport "${gpgSshSupportStr} > /dev/null";
+  + optionalString cfg.enableSshSupport ''
+    ${gpgSshSupportStr} > /dev/null
+  '';
 
   gpgFishInitStr = ''
     set -gx GPG_TTY (tty)
   ''
-  + optionalString cfg.enableSshSupport "${gpgSshSupportStr} > /dev/null";
+  + optionalString cfg.enableSshSupport ''
+    ${gpgSshSupportStr} > /dev/null
+  '';
```

Comment:
I'm not sure if this is necessary. In my `config.fish` it looks like the new line after `/dev/null` is added anyways.

```fish
     /nix/store/a3w53s6d2aj49m5sak2qr43snjm6f5d1-gnupg-2.4.8/bin/gpg-connect-agent --quiet updatestartuptty /bye
  > /dev/null
  ```